Add custom slippage tolerance

pierregee opened the pull request:
<!--  Thanks for sending a pull request! -->

#### What kind of PR is this?:
<!-- Use one of the following kinds:
/kind feature
/kind fix
/kind chore
/kind docs
/kind refactor
/kind dependencies
-->

/kind feature

#### What this PR does / why we need it:
- This will help the user set a custom slippage 

#### Which issue(s) does this PR fixes?:
<!--
(Optional) Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes #636

#### Additional comments?:

#### Developer Checklist:
<!--  
Merging into the main branch implies your code is ready for production. 
Before requesting for code review, please ensure that the following tasks 
are completed. Otherwise, keep the PR drafted.
-->

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

<!-- 
* If applicable 
-->

thedoublejay commented on mobile-app/app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx:
```diff
@@ -46,7 +46,7 @@ export function PoolSwapScreen ({ route }: Props): JSX.Element {
   const pairs = usePoolPairsAPI()
   const [poolpair, setPoolPair] = useState<PoolPairData>()
-  const [slippage, setSlippage] = useState(3)
+  const [slippage, setSlippage] = useState(new BigNumber(3))
```
@ivan-zynesis @pierregee We have it as 3% since our release. Since we don't have complaints for now, I suggest we keep it first.

thedoublejay reviewed (changes requested):
It doesn't scroll on iPhone SE when you click on Custom. Cannot view Submit

pierregee commented:
> It doesn't scroll on iPhone SE when you click on Custom. Cannot view Submit

@thedoublejay Updated! I added a ScrollView.

**Additional Info:** 
 - Android and iOS seems to be responding differently with `ScrollView` inside the `BottomSheet` so I added `KeyboardAvoidingView` to handle space at the top of the keyboard (tested with iPhone SE Simulator)
 - Opted out of using the `react-native-bottom-sheet/BottomSheetScrollView` because it has more erratic behaviour

See image below if `KeyboardAvoidingView` is NOT added (One Plus Nord 2)
![image](https://user-images.githubusercontent.com/13292662/139623849-e4c19b24-9eb4-4960-a3e8-6add10d50990.png)
